test/position-managers/IncreaseLiquidity.t.sol

```diff
@@ -164,6 +165,53 @@ contract IncreaseLiquidityTest is Test, PosmTestSetup, Fuzzers {
         assertApproxEqAbs(balance1BeforeAlice, currency1.balanceOf(alice), tolerance);
     }
 
+    function test_increaseLiquidity_withUnapprovedCaller() public {
+        // Alice and Bob provide liquidity on the range
+        // Alice uses her exact fees to increase liquidity (compounding)
```

Comment:
?? does she?

Reply:
i feel like this is quite a complicated test to just test something thats not as complicated?
might be easier to have just 1 person providing liquidity not 2.
Just like
- Call 1: Alice mints herself a position
- Call 2: Bob increases liquidity in alices position
No need for Bob to have a position in the same range that feels like its confusing it a bit